Sources/TuistCore/Protocols/CloudClienting.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+import RxSwift
+import TuistSupport
+
+public protocol CloudClienting {
```

Comment:
I'd place this file under `TuistCore/Cloud`, which should contain all the files that are specific to the Cloud framework. That makes it easier to spot to which framework the files in `TuistCore` are associated.

Reply:
sounds good.